x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/documentation.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DocLinksStart } from 'kibana/public';
+
+export class AsyncSearchIntroDocumentation {
+  private docsBasePath: string = '';
+
+  public setup(docs: DocLinksStart) {
```

Comment:
Nit: Having a `setup` method mimics the `plugin` interface, yet `setup` is called with `DocLinksStart` (which is from `start`, not `setup`). Maybe we could just have this be a class which accepts `DocLinksStart` in the `constructor` instead.